rclpy/rclpy/impl/rcutils_logger.py

```diff
@@ -257,6 +257,9 @@ def log(self, message, severity, **kwargs):
         The message will only be logged if its severity is not less than the severity threshold
         of the logger, and no logging filter causes the message to be skipped.
 
+        .. note::
+           Logging filters will only be evaluated if the message severity is not less than the
```

Comment:
> Logging filters will only be evaluated if the message severity is not less than the severity threshold of the logger.

Maybe it's because I'm not native, but I had to read this twice to understand what it meant.
Maybe having the negation earlier in the sentence (logging filters will *not* be evaluated if the message severity is less than...) or using "greater" rather than "is not less than" would be clearer.

Reply:
> Maybe it's because I'm not native, but I had to read this twice to understand what it meant.
> Maybe having the negation earlier in the sentence (logging filters will not be evaluated if the message severity is less than...) or using "greater" rather than "is not less than" would be clearer.

Agreed, the wording is a little difficult.  I personally prefer using "greater" for this sentence, but moving the "not" earlier in the sentence should also make it clearer.